test/hotspot/jtreg/runtime/logging/ProtectionDomainVerificationTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2021 :)

Reply:
I had a bug in my fix-copyrights script.